tests/test_live_validation.c

```diff
@@ -34,7 +34,7 @@ const struct test_validity_query queries[] = {{"93.175.146.0", 24, 12654, BGP_PF
 					      {"2001:7fb:ff03::", 48, 12654, BGP_PFXV_STATE_NOT_FOUND},
 					      {NULL, 0, 0, 0} };
 
-const int connection_timeout = 20;
+const int connection_timeout = 80;
```

Comment:
why was it necessary to increase the timeout?

Reply:
live tests (test_live_validation, test_dynamic_groups) failed for some jobs, because rpki cache instance was busy with other requests.